Assignment1/src/test/java/AssignmentMain.java

```diff
@@ -0,0 +1,107 @@
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.MatcherAssert.assertThat; 
+import static org.hamcrest.Matchers.*;
+import io.restassured.path.json.JsonPath;
+
+public class AssignmentMain {
+	
+	static String baseURI;
+	public static void main(String args[]) {
+		
+		// get users request getCall 1
+
+		baseURI = RestAssured.baseURI = "https://reqres.in/";
+		String list_users = given().queryParam("page", 2)
+		.when()
+		.get("/api/users?page=2")
+		.then().assertThat().statusCode(200)
+		.extract().response().asString();
+		
+		System.out.println("List users getCall 1");
+		System.out.println(list_users);
+		
+		// get single user getCall 2
+		int id;
+		id = 2;
+		
+		
+		String singleUser = given().queryParam("id", id)
+		.when()
+		.get("/api/users?page=2")
+		.then().assertThat().statusCode(200)
+		.extract().response().asString();
+		
+		System.out.println("I am single user getCall 2 ");
+		System.out.println(singleUser);
+		
+		
+		// single user not found getCall 3
+		int invalidId;
+		invalidId= 23;
+		String singleUserNotFound = given().queryParam("id", invalidId)
+		.when()
+		.get("/api/users?page=2")
+		.then().assertThat().statusCode(404)
+		.extract().response().asString();
+				
+		System.out.println("User not found getCall 3");
+		System.out.println(singleUserNotFound);
+		
+		// List Resource getCall 3
+		
+		String listResource = given()
+		.when()
+		.get("/api/unknown")
+		.then().assertThat().statusCode(200)
```

Comment:
All of these tests are not checking the response data. Please create pojo classes and then use them for validations.

Reply:
Created pojo classes and added some validations